test/langtools/tools/javac/api/TestJavacTaskWithWarning.java

```diff
@@ -50,7 +50,7 @@ public class TestJavacTaskWithWarning {
     public static void warningTest() throws Exception {
 
         // Create a source file that will generate a warning
-        String srcdir = System.getProperty("test.src");
+        String srcdir = System.getProperty(".");
```

Comment:
Sorry, I wasn't very precise, I mean something along these lines:
```suggestion
        String srcdir = ".";
```